src/rules/selector-max-specificity/README.md

```diff
@@ -0,0 +1,29 @@
+# selector-max-specificity
+
+Control the specificity of selectors.
```

Comment:
"Limit the specificity of selectors." - as an convention we use "Limit" to describe `max-*` rules.